Dockerfile

```diff
@@ -7,21 +7,20 @@ RUN pip install --upgrade pip
 WORKDIR /local
 COPY . /local
 
-LABEL maintainer="Khelil Sator <ksator@arista.com>"
-LABEL com.example.version="0.4.0"
-LABEL com.example.release-date="2023-02-16"
-LABEL com.example.version.is-production="False"
-
 ENV PYTHONPATH=/local
 RUN pip --no-cache-dir install .
 
-LABEL org.opencontainers.image.title ANTA
-LABEL org.opencontainers.artifact.description "network-test-automation in a Python package and Python scripts to test Arista devices."
-LABEL org.opencontainers.image.url https://www.anta.ninja
-LABEL org.opencontainers.image.documentation https://www.anta.ninja
-LABEL org.opencontainers.image.licenses APACHE2
-LABEL org.opencontainers.image.authors "Khelil Sator, Angélique Phillipps, Colin MacGiollaEáin, Matthieu Tache, Onur Gashi, Paul Lavelle, Guillaume Mulocher, Thomas Grimonet"
-LABEL org.opencontainers.image.base.name python
-LABEL org.opencontainers.image.source https://github.com/arista-netdevops-community/anta
+# Opencontainer labels
+LABEL   "org.opencontainers.image.title"="anta" \
+        "org.opencontainers.artifact.description"="network-test-automation in a Python package and Python scripts to test Arista devices." \
+        "org.opencontainers.image.source"="https://github.com/arista-netdevops-community/anta" \
+        "org.opencontainers.image.url"="https://www.anta.ninja" \
+        "org.opencontainers.image.documentation"="https://www.anta.ninja" \
+        "org.opencontainers.image.licenses"="Apache-2.0" \
+        "org.opencontainers.image.vendor"="The anta contributors." \
+        "org.opencontainers.image.authors"="Khelil Sator, Angélique Phillipps, Colin MacGiollaEáin, Matthieu Tache, Onur Gashi, Paul Lavelle, Guillaume Mulocher, Thomas Grimonet" \
+        "org.opencontainers.image.base.name"="python" \
+        "org.opencontainers.image.revision"="" \
```

Comment:
This should be set right?

Reply:
If everything works as expected, this field will be populated by the CI with:

```yaml
- name: Build and push
  uses: docker/build-push-action@v4
  with:
    context: .
    file: Dockerfile
    push: true
    platforms: linux/amd64,linux/arm64,linux/arm/v7
    tags: ${{ steps.meta.outputs.tags }}
    labels: ${{ steps.meta.outputs.labels }}
```